runtime/test/src/common.rs

```diff
@@ -0,0 +1,62 @@
+use ethabi::Contract;
+use graph::data::subgraph::*;
+use graph::prelude::*;
+use graph_chain_ethereum::DataSource;
+use semver::Version;
+use std::str::FromStr;
+use web3::types::Address;
+
+#[allow(unused)]
```

Comment:
Is this necessary? If the function is `pub` then it can never be considered unused.

Reply:
Yep, the compiler complains that it's unused when it doesn't have that annotation